File: Java/Assign4/src/Curvedbanner.java
package banner;

public class Curvedbanner extends Banner {
	private float radius;

	public Curvedbanner( float w, float h, float r) {
		super (w, h);
		radius = r;
	}
	public float getRadius() {
		return radius;
	}
	public void setRadius(float r) {
		radius = r;
	}
	public double area() {
		return super.area() - 0.86  * radius * radius;
	}
}

File: DotNet/Assignment 3/Program/CurvedBanner.cs
public class Curvedbanner : Banner 
{
    	private float radius;

	public Curvedbanner( float w, float h, float r) : base (w,h)
    {
		radius = r;
	}
	public double Radius { get; set; }

	public override  double area()
    {
		return base.area() - 0.86  * radius * radius;
	}
}
